front/utils/strings.util.ts

```diff
@@ -9,3 +9,13 @@ export const stringArrayIsNullOrEmpty = (
 ): boolean => {
   return str === null || str === undefined || str.length === 0;
 };
+
+export const validateEmail = (inputText: string): boolean => {
+  const mailformat = /^[\w-.]+@([\w-]+\.)+[\w-]+$/;
+  return !!mailformat.exec(inputText);
```

Comment:
On peut simplifier :

```suggestion
  return mailformat.test(inputText);
```